Marlin/src/lcd/menu/menu_media.cpp

```diff
@@ -148,7 +148,12 @@ void menu_media_filelist() {
     START_MENU();
     BACK_ITEM_F(TERN1(BROWSE_MEDIA_ON_INSERT, screen_history_depth) ? GET_TEXT_F(MSG_MAIN) : GET_TEXT_F(MSG_BACK));
     #if ENABLED(VOLUME_SD_ONBOARD)
-      ACTION_ITEM(MSG_SD_CARD, []{ card.changeMedia(&card.media_driver_sdcard); card.mount(); ui.goto_screen(menu_media_filelist); });
+      #if ENABLED(SDSUPPORT)
+        ACTION_ITEM(MSG_SD_CARD, []{ card.changeMedia(&card.media_driver_sdcard); card.mount(); ui.goto_screen(menu_media_filelist); });
+      #endif
+      #if ENABLED(SDIO_SUPPORT)
+        ACTION_ITEM(MSG_SDIO_CARD, []{ card.changeMedia(&card.media_driver_sdiocard); card.mount(); ui.goto_screen(menu_media_filelist); });
+      #endif
```

Comment:
Likewise, here it would be better to refer to "SD Card 1," "SD Card 2," or if it makes more sense, "SD Card (onboard)," "SD Card (external)," etc.

Reply:
> "SD Card (onboard)," "SD Card (external)," etc.

I think this option is better since `SD Card 1` wouldn't remind the user if `1` alone is onboard or external.